appdatabase/migrations/sql/1683627613_accounts_and_keycards_improvements.up.sql

```diff
@@ -0,0 +1,94 @@
+-- Possible `operable` column values:
+-- "no"        // an account is non operable it is not a keycard account and there is no keystore file for it and no keystore file for the address it is derived from
+-- "partially" // an account is partially operable if it is not a keycard account and there is created keystore file for the address it is derived from
+-- "fully"     // an account is fully operable if it is not a keycard account and there is a keystore file for it
+
+-- Adding new tables `keypairs` and `keypairs_accounts`
+CREATE TABLE IF NOT EXISTS keypairs (
+  key_uid VARCHAR PRIMARY KEY NOT NULL CHECK (length(trim(key_uid)) > 0),
+  name VARCHAR NOT NULL DEFAULT "",
+  type VARCHAR NOT NULL DEFAULT "",
+  derived_from VARCHAR NOT NULL DEFAULT "",
+  last_used_derivation_index INT NOT NULL DEFAULT 0,
+  synced_from VARCHAR NOT NULL DEFAULT "", -- keeps an info which device this keypair is added from
+  clock INT NOT NULL DEFAULT 0
+);
+
+CREATE TABLE IF NOT EXISTS keypairs_accounts (
+  address VARCHAR PRIMARY KEY,
+  key_uid VARCHAR,
+  pubkey VARCHAR,
+  path VARCHAR NOT NULL DEFAULT "",
+  name VARCHAR NOT NULL DEFAULT "",
+  color VARCHAR NOT NULL DEFAULT "",
+  emoji VARCHAR NOT NULL DEFAULT "",
+  wallet BOOL NOT NULL DEFAULT FALSE,
+  chat BOOL NOT NULL DEFAULT FALSE,
+  hidden BOOL NOT NULL DEFAULT FALSE,
+  operable VARCHAR NOT NULL DEFAULT "no", -- describes an account's operability (read an explanation at the top of this file)
```

Comment:
can we split `operable` into 3 separate conditional fields so that code will looks more readable? 
is_keycard_account bool 
exist_keystore_file bool
exist_keystore_file_derived_from bool

Reply:
I agree that this column in the db can store anything (since it's varchar), but on the code side there is `AccountOperable` type defined which has 3 predefined values and that type is used in the code, which is descriptive enough for usage. Adding those 3 columns I see as redundant and unnecessary thing in this case.